doc/header.tex

```diff
@@ -55,6 +55,18 @@
   keepspaces=true,
 }
 
+% An environment for describing variables in equations
+\newcommand*\wherelistlabel[1]{$#1 =$}
```

Comment:
Fancy new environment here.  I'm almost inclined to use `ensuremath` instead of the raw dollars, that way we don't end up with awkward syntax errors if someone tries to put this inside math mode or something.  Probably not worth it, but would like to know your opinion.  This is indeed going to be a nice addition if it is spread throughout the engineering ref code.

Reply:
Yes, `ensuremath` would be better and safer. I'll make that change.